packages/mds-compliance-engine/package.json

```diff
@@ -14,7 +14,7 @@
   ],
   "scripts": {
     "build": "tsc --build tsconfig.build.json",
-    "test": "DOTENV_CONFIG_PATH=../../.env nyc --use-spawn-wrap ts-mocha --project ../../tsconfig.json --exit",
+    "test": "DOTENV_CONFIG_PATH=../../.env jest --runInBand --forceExit",
```

Comment:
Is there no resolution for `--forceExit`?

Reply:
`--detechOpenHandles` doesnt offer any help, is a broken feature, as documented by open Issues on GH.